content/en/docs/ops/integrations/prometheus/index.md

```diff
@@ -0,0 +1,66 @@
+
+This configuration will set up scraping jobs for the control plane, as well as for all Envoy sidecars. Additionally, a job is configured to scrape application metrics for all pods with relevant `prometheus.io` annotations:
+
+* `prometheus.io/scrape` determines if a pod should be scraped. Set to `true` to enable scraping.
+* `prometheus.io/path` determines the path to scrape metrics at. Defaults to `/metrics`.
+* `prometheus.io/port` determines the port to scrape metrics at. Defaults to `80`.
+
+#### TLS settings
+
+The control plane, gateway, and Envoy sidecar metrics will all be scraped over plaintext. However, the application metrics will follow whatever Istio configuration has been configured for the workload. In particular, if [Strict mTLS](/docs/tasks/security/authentication/authn-policy/#globally-enabling-istio-mutual-tls-in-strict-mode) is enabled, then Prometheus will need to be configured to scrape using Istio certificates.
+
+When using the bundled Prometheus deployment, this is configured by default. For external integrations, please follow [Provision a certificate and key for an application without sidecars](/blog/2020/proxy-cert/) to provision a certificate for Prometheus, then add the TLS scraping configuration.
+
+### Option 2: Metrics merging
+
+{{< warning >}}
+This option is newly introduced in Istio 1.6 and is considered `alpha` at this time.
+{{< /warning >}}
+
+To simplify configuration, Istio has the ability to control scraping entirely by `prometheus.io` annotations. This allows Istio scraping to work out of the box with standard configurations such as the ones provided by the [Helm `stable/prometheus`](https://github.com/helm/charts/tree/master/stable/prometheus) charts.
+
+{{< tip >}}
+While `prometheus.io` annotations are not a core part of Prometheus, they have become the de facto standard to configure scraping.
+{{< /tip >}}
+
+To enable this setting, pass `--set meshConfig.enablePrometheusMerge=true` during [installation](/docs/setup/install/istioctl/). When this setting is enabled, appropriate `prometheus.io` annotations will be added to all workloads to set up scraping. If these annotations already exists, they will be overwritten. In these case, the Envoy sidecar will merge Istio's metrics with the application metrics.
+
+When using this option, all metrics will be exposed over plain text.
+
+This feature may not be suitable if:
+* You need to scrape metrics using TLS
+* Your application exposes metrics with the same names as Istio metrics. For example, your application metrics expose an `istio_requests_total` metric. This may happen if the application is itself running Envoy.
```

Comment:
```suggestion
* Your application exposes metrics with the same names as Istio metrics. For example, your application metrics expose an `istio_requests_total` metric. This might happen if the application is itself running Envoy.
```